WowCharComparerWebApp/Controllers/User/LoginController.cs

```diff
@@ -8,5 +14,86 @@ public IActionResult Index()
         {
             return View();
         }
+
+        [Route("Login/PasswordRecovery")]
+        [HttpGet]
+        public IActionResult PasswordRecoveryView()
+        {
+            return View("PasswordRecovery");
+        }
+
+        [Route("Login/PasswordRecovery/Confirmation")]
+        public IActionResult PasswordResetConfirmationView(string userID)
+        {
+            ViewBag.Message = userID;
+            return View("PasswordChanged");
+        }
+
+        [HttpPost]
+        public IActionResult PasswordChanged(string newPassword, string newPasswordConfirmation, Guid userID)
+        {
+            List<string> passwordsToCheck = new List<string>();
```

Comment:
I think there is no need to use List to store only 2 string (because it will be always only 2 strings). List is useful if we are gonna do something else with collection ex. add, delete, sort etc.